modules/ting_search/plugins/content_types/search_backends.inc

```diff
@@ -51,7 +51,7 @@ function ting_search_backend_engines_content_type_render($subtype, $conf, $panel
 
     // Create the option as a link.
     $txt = format_string('Search @backend', array('@backend' => $backend['title']));
-    $title = l($txt, $path, array('query' => $conditions));
+    $title = l(t($txt), $path, array('query' => $conditions));
```

Comment:
Scrutinizer pukes on t($txt). The Drupal documentation on t() also warns against it - although format_string() fix that.
However, I think the issue lies elsewhere... 
At installation,  "Search @backend" will evaluate to "Search ".
"Search Materialer" and "Search Content" won't be added to the database of strings to be translated until the code is run. 
I suggest we add the strings to the database of strings to be translated, at installation.
So, add t("Search Materialer") and t("Search Content") to hook_install() (and a hook_update() for good measure)

Reply:
But the"Search Materialer" and "Search Content" is already in da.po translation file, so wont they get added during installation and update? Do they really need to be in the database first to be imported?

Reply:
hmm... I think you're right. I can't find a good way to get around translating variables.

Reply:
This has annoyed me as well several times before. I think that maybe we have misused the translation system a bit in DDB CMS.